batchconsumer/batchermanager.go

```diff
@@ -184,7 +186,7 @@ func (b *batcherManager) startMessageHandler(
 
 					batcher.AddMessage(tmp.msg, tmp.pair)
 				} else if err != nil {
-					b.log.ErrorD("add-message", kv.M{
+					lg.ErrorD("add-message", kv.M{
 						"err": err.Error(), "msg": string(tmp.msg), "tag": tmp.tag,
```

Comment:
@xavi- is there danger in re-logging `tmp.msg` here? If so I can change this to omit `tmp.msg` and add a call to log it to the file

Reply:
Ahh yeah, good catch.  Yeah, I agree, it makes sense to omit `tmp.msg` from the stderr log and move it to the file log.